extension/WebJobs.Extensions.RabbitMQ.Tests/Trigger/RabbitMQListenerTests.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Moved file `extension/WebJobs.Extensions.RabbitMQ.Tests\RabbitMQListnerTests.cs` to `extension/WebJobs.Extensions.RabbitMQ.Tests\Trigger\RabbitMQListnerTests.cs` but also replaces all tests.

The previously commented out tests that used to test for retry logic are not added, since the implementation for mocking classes turned to be complicating both test file and the Listener class.